packages/ui/ToggleItem/ToggleItem.js

```diff
@@ -15,26 +15,69 @@ import kind from '@enact/core/kind';
 import EnactPropTypes from '@enact/core/internal/prop-types';
 import PropTypes from 'prop-types';
 import compose from 'ramda/src/compose';
-import React from 'react';
+import React, {Fragment} from 'react';
 
 import ComponentOverride from '../ComponentOverride';
 import ForwardRef from '../ForwardRef';
+import Slottable from '../Slottable';
 import Toggleable from '../Toggleable';
 import Touchable from '../Touchable';
 
 import componentCss from './ToggleItem.module.less';
 
+const ItemIcon = kind({
+	name: 'ItemIcon',
+	propTypes: {
+		/**
+		 * The `itemIcon` to render in this item.
+		 *
+		 * @type {Node}
+		 * @public
+		 */
+		itemIcon: PropTypes.node
+	},
+	render: ({itemIcon}) => {
+		return (
+			<ComponentOverride component={itemIcon} />
+		);
+	}
+});
+
 // eslint-disable-next-line
-const iconCreator = (position) => ({disabled, icon, iconComponent, selected, iconPosition}) => {
```

Comment:
Let's not disable eslint entirely!  Only disable the rule needed here.

Reply:
Updated lint rule